docs/chapters/traversal/index.md

```diff
@@ -0,0 +1,178 @@
+# Understand Jina Recursive Document Representation
+
+In Jina, each `Document` is represented as a recursive representation (tree).
+
+A rooted recursive representation has a root node and every node has X children.
+In Jina, the root node is the document itself, while the *left* & *right* children are referred to as *chunks* and *matches* respectively.
+
+![rooted-binary-tree](img/rooted-binary-tree.png)
+
+The above image illustrates a minimal document structure: A document (root node) consists of two child nodes: *chunks* and *matches*.
+`chunks`s are a sequence of `Document`s which is attached to the root document with a higher `granularity` degree. `Matches` is a sequence of `Document`s  which are semantically related to the root document.
+We'll dive into these concepts in this document:
```

Comment:
Maybe use "this chapter" instead of "this document" because people might confuse it with Jina's Document terminology.

Reply:
done